### PR TITLE
Avoid RIF Data Vault dependency

### PR DESCRIPTION
**Pull request by diegograsso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,10 +124,62 @@ The following methods are confirmed to work: `eth_chainId`, `eth_accounts`, `net
...
 
-- Permissioned apps: for example, apps using Google OAuth to receive user's email are categorized in this flavor. This process of requestion credentials to grant user access is common and usually relies on centralized data silos. This dApp flavor will cover requesting user's Verifiable Credentials in a fully user-centric manner - this is setup in the backend activating _Selective disclosure requests_
+- Permissioned apps: for example, apps using Google OAuth to receive user's email are categorized in this flavor. This process of requesting credentials to grant user access is common and usually relies on centralized data silos. This dApp flavor will cover requesting user's Verifiable Credentials in a fully user-centric manner - this is setup in the backend activating _Selective disclosure requests_
 
 - Closed apps: for example, a back office. This are apps that only specific user's can access. This flavor is used to prove the user accessing an app holds or is delegated by a specific identity - perform this validations in your server's business logic
 
+### Set up for the different flavors
+- Fully-decentralized apps:
```
Comment (ilanolkies) [inline README.md:132]:
```suggestion
- Open apps: 
```

File: README.md [hunk 2]
```diff
@@ -124,10 +124,62 @@ The following methods are confirmed to work: `eth_chainId`, `eth_accounts`, `net
...
+```typescript
+  import RLogin from '@rsksmart/rlogin'
+ 
+  export const rLogin = new RLogin({
+    cachedProvider: false,
+    providerOptions: {
+      // providers configurations
+    },
+    supportedChains: [1, 30, 31],
+    backendUrl: 'http://url-to-backend'  // just add the backend url
```
Comment (ilanolkies) [inline README.md:154]: I would only describe how to add this key to the config

File: README.md [hunk 3]
```diff
@@ -124,10 +124,62 @@ The following methods are confirmed to work: `eth_chainId`, `eth_accounts`, `net
...
+
+  export const rLogin = new RLogin({
+    cachedProvider: false,
+    providerOptions: {
+      // providers configurations
+    },
+    supportedChains: [1, 30, 31],
+    backendUrl: 'http://url-to-backend',
+     // add the modules that will handle the data vault connection and the service url for the supported networks
+    dataVaultOptions: {
```
Comment (ilanolkies) [inline README.md:171]: Same here. I would describe the new key.

also, can we export a factory method that does all this?

**Review by ilanolkies — CHANGES_REQUESTED** [review]
LRGTM

**Review by ilanolkies — APPROVED** [review]
Great PR!

File: README.md [hunk 4]
```diff
@@ -124,10 +124,59 @@ The following methods are confirmed to work: `eth_chainId`, `eth_accounts`, `net
...
+      package: RIFDataVault,
+      serviceUrl: {
+        30: 'https://data-vault.identity.rifos.org',
```
Comment (jessgusclark) [inline README.md:174]: Since this example supports chains 1, 30, and 31, so the serviceUrl should also support those chains.

**Review by jessgusclark — COMMENTED** [review]
Tested this with the Permissioned Sample app using `yarn link` and it worked as described. My only comment is regarding the readme which throws a 404 error if the URL is not set for a specific chain.

File: README.md (hunk 4 above)
Comment (ilanolkies) [reply README.md:174]: Isn’t it better to use a single url? We are handling network via did method suffix

Comment (jessgusclark) [reply README.md:174]: @ilanolkies I agree so then the code change should be like this:

```
serviceUrl: 'https://....'
```

And not take multiple endpoints.

**Pull request merged**